### PR TITLE
feat(core): added show and hide delay to tooltip and hint directives

### LINKED ISSUE #202: [FEATURE] PointerHint add Delay Input
# 🚀 Feature request

Add a delay input for the PointerHint, so that it can be used similiar to a tooltip, but following the cursor
https://taiga-ui.dev/tui-pointer-hint

### PR DESCRIPTION
**Pull request by fynnfeldpausch**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #202 

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Review by MarsiBarsi — APPROVED** [review]

File: projects/core/components/tooltip/tooltip.component.ts [hunk 1]
```diff
@@ -33,6 +33,14 @@ export class TuiTooltipComponent {
     @tuiDefaultProp()
     direction: TuiDirection = 'bottom-left';
 
+    @Input()
+    @tuiDefaultProp()
+    showDelay = 0;
```
Comment (waterplea) [inline projects/core/components/tooltip/tooltip.component.ts:38]: This should have default values, 500 and 200

**Review by waterplea — CHANGES_REQUESTED** [review]
Sorry for the delay, please take a look at the comment and we're good to merge.

**Comment by KarimovDev** [comment]
Hi, @fynnfeldpausch! 
Do you need some help here? I can push the PR includes this feature and fixes above

**Comment by waterplea** [comment]
> Hi, @fynnfeldpausch!
> Do you need some help here? I can push the PR includes this feature and fixes above

As far as I know @fynnfeldpausch got pulled to another project, unrelated to Taiga UI so he paused all his contributions. Feel free to pick it up, @KarimovDev if that issue is relevant to you.

**Pull request closed without merging**